3574/1/option_parser.rb

```diff
@@ -0,0 +1,27 @@
+class Options
+  def call(from_file)
+    options_parser(from_file)
+  end
+
+  private
+
+  def number(from_file, opts)
+    opts.on('--top NUMBER') { |opt| from_file[:number] = opt }
+  end
+
+  def gemname(from_file, oopts)
```

Comment:
please use underscored names for variables and method names according to styleguide